Formula 1/ViewModels/CircuitsViewModel.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  CircuitsViewModel.swift
+//  Formula 1
+//
+//  Created by Gio on 11/13/19.
+//  Copyright © 2019 Gio. All rights reserved.
+//
+
+import Foundation
+
+class CircuitsViewModel: ObservableObject {
+    @Published private var circuits = [Circuit]()
+    
+     weak var delegate: Fetchable?
+    
+    var numberOfCircuits: Int { circuits.count }
+    
+    init() {
+           fetch()
+       }
+       
+    private func fetch() {
```

Comment:
This will only get called once and never again. You probably want to fetch again.

Reply:
Can you clarify

Reply:
fetch() is only called on init, which only happens once. Which only happens when the VC is initiallized (once). Once your app launches cold that's the data you'll ever have. You won't ever get new data unless you kill the app.

Reply:
What should i do for now?

Reply:
that's up to you... Is this the behavior you want for your app?

Reply:
I'd like to eventually implement a swipe-down gesture to refresh the table-view